scripts/storagedrawers.zs

```diff
@@ -67,12 +68,12 @@ for i in 0 to 6 {
 	recipes.addShaped("gregified_fulldrawer1_" + woods[i], <storagedrawers:basicdrawers:0>.withTag(woodStrings[i]),
```

Comment:
Do you mind changing the recipe names to be the "standard" format? Makes potential future debugging or recipe changes easier. You don't have to do this for right now though.